package.json

```diff
@@ -80,9 +80,9 @@
     "show-linked-packages": "find node_modules node_modules/@* -depth 1 -type l -print",
     "link-packages": "./scripts/link-packages.sh",
     "unlink-packages": "find node_modules node_modules/@* -depth 1 -type l -print | awk -F/ '{ printf \"%s/%s\\n\",$2,$3; }' | xargs yarn unlink && yarn install --force",
-    "build": "NODE_OPTIONS=--max-old-space-size=4096 IMAGE_INLINE_SIZE_LIMIT=0 INLINE_RUNTIME_CHUNK=false REACT_APP_HIDE_SPLASH=false DISABLE_ESLINT_PLUGIN=true react-app-rewired build && node headers.js",
-    "dev": "react-app-rewired start",
-    "dev:silent": "BROWSER=none react-app-rewired start",
+    "build": "NODE_OPTIONS=--max-old-space-size=4096 ESLINT_NO_DEV_ERRORS=true IMAGE_INLINE_SIZE_LIMIT=0 INLINE_RUNTIME_CHUNK=false REACT_APP_HIDE_SPLASH=false DISABLE_ESLINT_PLUGIN=true react-app-rewired build && node headers.js",
+    "dev": "NODE_OPTIONS=--max-old-space-size=4096 ESLINT_NO_DEV_ERRORS=true IMAGE_INLINE_SIZE_LIMIT=0 INLINE_RUNTIME_CHUNK=false REACT_APP_HIDE_SPLASH=false react-app-rewired start",
+    "dev:silent": "NODE_OPTIONS=--max-old-space-size=4096 ESLINT_NO_DEV_ERRORS=true IMAGE_INLINE_SIZE_LIMIT=0 INLINE_RUNTIME_CHUNK=false REACT_APP_HIDE_SPLASH=false BROWSER=none react-app-rewired start",
```

Comment:
what about making something in `scripts` that applies all the required environment variables and calling that script here?

Reply:
I'd love to, but it seems like writing such a script right might require a bit more thought. This is indirectly needed for the CRA5 upgrade, so I didn't want to touch too much at once.

Reply:
that's reasonable.